modules/stochastic_tools/test/tests/reporters/ActiveLearningGP/Main_adam.i

```diff
@@ -0,0 +1,123 @@
+[StochasticTools]
```

Comment:
To follow the convention in MOOSE, try not using capital letters in the test input file names.